include/zephyr/drivers/sensor/rrh46410.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_SENSOR_RRH46410_H_
+#define ZEPHYR_INCLUDE_DRIVERS_SENSOR_RRH46410_H_
+
+#include <zephyr/drivers/sensor.h>
+
+enum sensor_channel_rrh46410 {
+	/** 1 byte Indoor Air Quality */
+	SENSOR_CHAN_IAQ = SENSOR_CHAN_PRIV_START,
+	/** 2 bytes Total Volatile Organic Compounds */
+	SENSOR_CHAN_TVOC,
+	/**  2 bytes Ethanol Equivalent */
+	SENSOR_CHAN_ETOH,
+	/** 2 bytes Estimated CO2 */
+	SENSOR_CHAN_ECO2,
+	/** 1 byte Relative IAQ */
+	SENSOR_CHAN_RELIAQ,
```

Comment:
These custom enums need to have the sensor name in them. e.g., `SENSOR_CHAN_RRH46410_IAQ`